### PR TITLE
 Make home URL optional and support dependencies

### PR DESCRIPTION
**Pull request by minhdanh**
**What this PR does / why we need it**:
`home` is optional according to Helm's docs: https://helm.sh/docs/topics/charts/
```
home: The URL of this projects home page (optional)
```

Chart.yaml also has optional declaration for dependencies:
```
dependencies: # A list of the chart requirements (optional)
  - name: The name of the chart (nginx)
    version: The version of the chart ("1.2.3")
    repository: The repository URL ("https://example.com/charts") or alias ("@repo-name")
    condition: (optional) A yaml path that resolves to a boolean, used for enabling/disabling charts (e.g. subchart1.enabled )
    tags: # (optional)
      - Tags can be used to group charts for enabling/disabling together
    enabled: (optional) Enabled bool determines if chart should be loaded
    import-values: # (optional)
      - ImportValues holds the mapping of source values to parent key to be imported. Each item can be a string or pair of child/parent sublist items.
    alias: (optional) Alias to be used for the chart. Useful when you have to add the same chart multiple times
```

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: etc/chart_schema.yaml [hunk 1]
```diff
@@ -20,3 +21,17 @@ maintainer:
...
   email: str(required=False)
   url: str(required=False)
+---
+dependency:
+  name: str()
+  version: str()
+  repository: str()
+  condition: str(required=False)
+  tags: list(str(), required=False)
+  enabled: bool()
```
Comment (smlx) [inline etc/chart_schema.yaml:31]: This field is optional according to https://helm.sh/docs/topics/charts/#the-chartyaml-file

Comment (minhdanh) [reply etc/chart_schema.yaml]: @smlx Thanks. I've just updated that

**Comment by minhdanh** [comment]
@smlx Can we have this merged this soon?

**Review by smlx — APPROVED** [review]
I'd love to, but I don't have a commit bit. That said, LGTM.

**Review by unguiculus — APPROVED** [review]

**Pull request merged**